packages/docs/getting-started/internationalization.md

```diff
@@ -224,3 +224,25 @@ export default {
   },
 };
 ```
+## Page reload on language change
+
+By default, the page is reloading while changing language.
+
+To prevent this, you can set `reloadOnLanguageChange: false`  in the `i18n` configartion object in the `nuxt.config.js`.
```

Comment:
```suggestion
By default, the page is reloading after changing the language. To prevent this, you can set `reloadOnLanguageChange: false`  in the `i18n` configuration object in the `nuxt.config.js`.
```